ocaml/xapi/cert_distrib.mli

```diff
@@ -0,0 +1,15 @@
+type existing_cert_strategy = Erase | Merge
```

Comment:
@psafont  I added this - it could be useful for pool join:

```
Cert_distrib.go ~from_hosts:existing_pool_hosts ~to_hosts:[joining_host] ~existing_cert_strategy:Erase ;
Cert_distrib.go ~from_hosts:[joining_host] ~to_hosts:existing_pool_hosts ~existing_cert_strategy:Merge ;
```

but maybe we only want to distribute a joiner's cert to other slaves once it has 'officially' joined?

Reply:
We want to distribute certificates before the joiner restarts itself, there's no way to know if the host has just joined a pool after that point, so the filesystems must be correct before it restarts. The database will be refreshed when it sees its internal certificate is missing from the db.